src/Workspaces/Core/Portable/Execution/WellKnownSynchronizationKinds.cs

```diff
@@ -3,33 +3,48 @@
 namespace Microsoft.CodeAnalysis.Serialization
 {
     // TODO: Kind might not actually needed. see whether we can get rid of this
-    internal static class WellKnownSynchronizationKinds
+    internal enum WellKnownSynchronizationKinds
```

Comment:
:bulb: This should end with `Kind` instead of `Kinds` (the latter would be for a `[Flags]` enum).